internal/test/helpers.go

```diff
@@ -122,6 +122,11 @@ type UsersClient struct {
 	Store map[string]*pb.User
 }
 
+func (c *UsersClient) GetUserByEthAddr(_ context.Context, _ *pb.GetUserByEthRequest, _ ...grpc.CallOption) (*pb.User, error) {
+	//TODO implement me
+	panic("implement me")
```

Comment:
is this ok like this, or just so interface contract works?

Reply:
yeah, this is so interface contracts work.